cmd/chisel/cmd_cut.go

```diff
@@ -57,6 +62,27 @@ func (cmd *cmdCut) Execute(args []string) error {
 		return err
 	}
 
+	maintenance := archive.Standard
+	if time.Now().After(release.Maintenance.EndOfLife) {
+		maintenance = archive.Unmaintained
+		if cmd.Ignore == "unmaintained" {
+			logf(`Warning: This release has reached the "unmaintained" maintenance status. ` +
+				`See https://documentation.ubuntu.com/chisel/en/latest/reference/chisel-releases/chisel.yaml/#maintenance to be safe`)
+		} else {
+			return fmt.Errorf(`this release has reached the "unmaintained" maintenance status, ` +
+				`see https://documentation.ubuntu.com/chisel/en/latest/reference/chisel-releases/chisel.yaml/#maintenance for details`)
+		}
+	} else if time.Now().Before(release.Maintenance.Standard) {
```

Comment:
What about the other possibilities?  (expanded but not using the right archive, etc)